.monorepo.json

```diff
@@ -1,4 +1,8 @@
 {
+  "versions": {
+    "stable": "8.0.0-beta.5",
```

Comment:
FYI on Angular we determine the version based on git tags; see https://github.com/angular/angular/blob/master/tools/bazel_stamp_vars.js

If we also had a scheme for making the 0.* versions based on the stable ones, then we'd only need one bit to describe the versioning.

That would help us move to bazel for build/publish someday

Reply:
All done.